frontend/awx/access/users/Users.tsx

```diff
@@ -203,15 +203,15 @@ export function AccessTable(props: { url: string }) {
         variant: ButtonVariant.primary,
         icon: PlusIcon,
         label: t('Add users'),
-        shortLabel: t('Add'),
+        shortLabel: t('Add Access'),
```

Comment:
The `AccessTable` component is not used since we added the new `ResourceAccessList` component. I think I forgot to clean this one up.